docs/notes/2.31.x.md

```diff
@@ -42,6 +42,18 @@ Added the `[python].default_to_resolve_interpreter_constraints` option which whe
 
 Go now compiles with trimpath to strip sandbox paths from output, allowing for reproducible builds.
 
+#### NEW: nFPM Native Libs
```

Comment:
Wondering: why is this a separate backend and not a new feature in the existing one?